doc/scheduler-scripting.tex

```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
 
 \end{enumerate}
 
-\begin{figure}[htbp]
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty1}}
+		\includegraphics[scale=0.55]{images/putty1}
 	\caption{SSH tunnel configuration 1}
 	\label{fig:putty1}
 \end{figure}
 
-\begin{figure}[htbp]
+
+
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty2}}
+		\includegraphics[scale=0.90]{images/putty2}
```

Comment:
Same I think scale is an issue for HTML render

Reply:
fixed